shared/ui-components/src/page_components/UserNav.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from "react"
+import "./UserNav.scss"
+import { LocalizedLink } from "../atoms/LocalizedLink"
+import { t } from "../helpers/translator"
+import { NavbarDropdown } from "../headers/SiteHeader/SiteHeader"
+
+export interface UserNavProps {
+  signedIn: boolean
+  children: JSX.Element | JSX.Element[]
```

Comment:
Let's avoid the use of `JSX.Element` per https://github.com/bloom-housing/bloom/issues/450